core/modules/parser/QSMySqlListener.cc

```diff
@@ -847,6 +888,11 @@ class SelectElementsAdapter :
         SelectListFactory::addValueExpr(_selectList, valueExpr);
     }
 
+    void checkContext() const override {
+        // optional:
+        // antlr4::Token *star;
```

Comment:
Why leave the commented code here? Is it planned to be used?

Reply:
for this and the rest; see the description of checkContext.